connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -57,6 +57,10 @@
     public static final String VALUE_CONVERTER_CLASS_DOC =
             "Converter class for value Connect data that implements the <code>Converter</code> interface.";
 
+    public static final String VALUE_CONVERTER_SCHEMA_CACHE_SIZE_CONFIG = "value.converter.schemas.cache.size";
```

Comment:
Yeah, I agree its not ideal. Unfortunately the way Kafka normally does configs does not leave room for a better solution where the different configuration sets can be clearly separated so its clear where you need to look for config options. And agree its not great having to look in multiple locations, but that is the reality of a pluggable system.